pybrake/queries.py

```diff
@@ -10,7 +10,13 @@
 
 
 class QueryStat(TDigestStat):
-    __slots__ = TDigestStat.__slots__ + ("query", "method", "route", "time")
```

Comment:
What is the violation here? The new version is more verbose?

Reply:
It gives below error

```
************* Module pybrake.queries

pybrake/queries.py:7:0: W0244: Redefined slots ['count', 'sum', 'sumsq', 'td', 'tdigest'] in subclass (redefined-slots-in-subclass)
```